wdl2cwl/tests/cwl_files/hisat2_1.cwl

```diff
@@ -66,8 +63,8 @@ requirements:
             hisat2 \
             -p $(inputs.threads) \
             -x $(inputs.indexFiles[0]) \
-            $(inputs["inputR2"] === "" ? "-U": "-1") $(inputs.inputR1.path) \
-            -2$(inputs.inputR2) \
+            $(inputs["inputR2"].path === null ? "-U": "-1") $(inputs.inputR1.path) \
+            $(inputs["inputR2"].path === null ? "" : "-2" + inputs["inputR2"].path ) \
```

Comment:
```suggestion
            $(inputs["inputR2"] === null ? "-U": "-1") $(inputs.inputR1.path) \
            $(inputs["inputR2"] === null ? "" : "-2" + inputs["inputR2"].path ) \
```
If `inputR2` is not provided then `inputs["inputR2"]` will be `null` and the reference `inputs["inputR2"].path` will make an error